notebooks/article_1.py

```diff
@@ -0,0 +1,1142 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Data Insight: Top Factors For Startup Success
+# 
+# Building a successful startup is hard. Statistics show that [90% of startups fail](https://www.failory.com/blog/startup-failure-rate). So, what are the main factors that make successful startups?
+# 
+# In this article, we will explore the [Startup Success Prediction dataset](https://www.kaggle.com/datasets/manishkc06/startup-success-prediction) released on [Kaggle](https://www.kaggle.com/) to find data insights on the success of startups.
+# 
+# This dataset contains information about industries, acquisitions, and investment details of nearly a thousand startups based in the USA from the 1980s to 2013.
+# 
+# To be able to extract interesting trends from data, it's essential to understand how to transform raw data into meaningful information. To achieve this goal, we will perform the following steps: 
+# 1. Data exploration
+# 2. Pre-processing 
+# 3. Data analysis 
+# 4. Data-driven conclusions
+# 
+# Here are the main questions we will answer:
+# - In which industries are startup acquisitions most common?
```

Comment:
Did you answer these questions in the article?
If not, change the bullet points (or remove them).